scripts/esReIndex.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from managers import ElasticsearchManager
+from main import loadEnvFile
+
+def main():
+
+    '''Re-indexing works into new ES cluster'''
+
+    loadEnvFile('local-qa', fileString='config/{}.yaml')
+
+    esManager = ElasticsearchManager()
+    esManager.createElasticConnection()
+    esManager.createElasticSearchIndex()
+
```

Comment:
Import `ESWork` and run `ESWork.init()` here

Reply:
When I put ESWork.init() at line 15 and run the script I get this ValueError.

raise ValueError("You cannot perform API calls on the default index.")
ValueError: You cannot perform API calls on the default index.

Reply:
oh, I know why this is failing, the `scripts/__init__.py` file is causing the import to happen first still. That traces back to `import scripts` from `scriptRunner.py`. To get this to work correctly we'd have to set the environment variables _before_ that import line, which is possible but a bigger refactor job. Would actually be a general improvement to prevent this kind of thing in the future.

Reply:
Thanks for the suggestion. I just did that and now the `drb_dcdw_qa_new_cluster` mapping is now filled out when I run the script.